game.cpp

```diff
@@ -1375,7 +1404,19 @@ void test_game() //!OCLINT tests may be many
     assert(!has_any_player_food_collision(g));
   }
 
-  // #392: When a player gets within the radius of food it eats it
+#ifdef FIX_ISSUE_621
+  // (621) A player that is out does not eat food
+  {
+    game g;
+    put_player_on_food(g.get_player(0), g.get_food()[0]);
```

Comment:
This works fine! Maybe this can be made more expressive by using a simple function called `put_first_player_on_first_food`, but you decide here :-)

Reply:
Nah, I don't feel its is useful to add a function just for that one test :)